### PR TITLE
Added a transform script which cleans the data

### PR DESCRIPTION
**Pull request by caithomas6789**
- Added a transform script which cleans some of the tag data up.
- Added a genre csv file which includes some common genres which frequently appear.

### CODE REVIEW

File: genres.csv [hunk 1]
```diff
@@ -0,0 +1,23 @@
+Metal
```
Comment (IshikaM19) [inline genres.csv:1]: Nice that you've created a separate file for this

File: pipeline/transform.py [hunk 2]
```diff
@@ -0,0 +1,66 @@
+"""
+Script to clean and transform all the data from the extract script.
+"""
+import pandas as pd
+
+DNB = ['Drum & Bass', 'Dnb', 'Drum N Bass']
```
Comment (IshikaM19) [inline pipeline/transform.py:7]: Nice

File: pipeline/transform.py [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+    Converts a list of dictionaries into a pandas dataframe and returns it.
+    """
+    return pd.DataFrame(extracted_data)
+
+
+def clean_tags(tags: list[str]) -> list[str]:
+    """
+    Cleans the tags associated with the album / track.
+    """
+    tags_set = set()
```
Comment (IB64) [inline pipeline/transform.py:23]: Nice use of set to remove duplicates

**Review by IB64 — APPROVED** [review]
nice

**Pull request merged**